deps/windows-install.py

```diff
@@ -3,7 +3,7 @@
 import json
 with open('../package.json') as f:
     librdkafkaVersion = json.load(f)['librdkafka']
-librdkafkaWinSufix = '7' if librdkafkaVersion == '0.11.5' else '';
+librdkafkaWinSufix = '7' if librdkafkaVersion == '1.1.0' else '';
```

Comment:
Why ?

Reply:
if you revert this diff, CI should be ✅

Reply:
That’s a good question, I assumed the default should be changed regardless. 

I just got off a 12.5 hour flight, so certainly missing a few things! 😅

Reply:
`librdkafkaWinSufix` looks to be only for `dllPath` & `libPath` maybe this variable can removed and dllPath&libPath  updated, because we won't upgrade to `0.11.5` 😅

Reply:
@GeoffreyHervet I’ve reverted this diff ✅